.github/workflows/check-code-quality.yml

```diff
@@ -29,7 +29,7 @@ concurrency:
 jobs:
   check-code-quality:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
We don't need to do that. Just use `ubuntu-latest`. It is already default `24.04`.

Reply:
I intentionally did this because `ubuntu-latest` is still migrating to 24.04 (see [**target date**](https://github.com/actions/runner-images/issues/10636)) and even though we had `ubuntu-latest` by default, we were running on 22.04 as you can see in this [example](https://github.com/PowerGridModel/power-grid-model/actions/runs/11234754635/job/31231229579).